.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: stable
+    rev: 22.3.0
```

Comment:
Is this an intentional change? Asking because it doesn't look like there's anything in the PR that seems like it might depend on pre-commit version.

Reply:
Ah, yeah - the hook was just broken for me so I pinned it to a version that worked :sweat_smile: